2020-08-20 17:23:27,784 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2020-08-20 17:23:27,784 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python38\lib\site-packages\instabot\bot\bot.py
2020-08-20 17:23:27,784 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2020-08-20 17:23:28,085 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2020-08-20 17:23:28,353 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-08-20 17:23:28,607 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-08-20 17:23:28,871 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2020-08-20 17:23:31,236 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2020-08-20 17:23:31,236 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "The password you entered is incorrect. Please try again.", "invalid_credentials": true, "error_title": "Incorrect password for 1", "buttons": [{"title": "Try Again", "action": "dismiss"}], "status": "fail", "error_type": "bad_password"}'
2020-08-20 17:23:31,237 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-08-20 17:23:31,237 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: The password you entered is incorrect. Please try again.
2020-08-20 17:23:31,237 - instabot version: 0.117.0 (api) - INFO - Error type: bad_password
2020-08-20 17:23:31,237 - instabot version: 0.117.0 (api) - ERROR - Failed to login go to instagram and change your password
2020-08-20 17:23:31,237 - instabot version: 0.117.0 (api) - INFO - Username or password is incorrect.
